docs/index.md

```diff
@@ -38,41 +38,39 @@ resource "ec_deployment" "my_deployment" {
 
 ## Authentication
 
-The Elastic Cloud provider offers two methods of authentication against the remote API. Depending on the target environment, one or all can be used. By default the `endpoint` which the provider will target is the Public API of the Elasticsearch Service (ESS) offering.
+The Elastic Cloud (EC) provider offers two methods of authentication against the remote API: `apikey` or a combination of `username` and `password`. Depending on the environment, one or both can be used. The Public API of Elasticsearch Service (ESS) is the default `endpoint` that the provider will target.
 
-Only one of `apikey` or `username` and `password`  can be specified at one time. The Elasticsearch Service (ESS) offering, only supports API Keys as the authentication mechanism. When targeting an ECE
-Installation, `username` and `password` can be used.
+Elasticsearch Service (ESS) only supports `apikey`. Elastic Cloud Enterprise (ECE) supports both `apikey` and a combination of `username` and `password`.
 
 !> **Warning:** Hard-coding credentials into any Terraform configuration is not
 recommended, and risks secret leakage should this file ever be committed to a
 public version control system.
 
-### API Key authentication (Recommended)
+### API Key authentication (recommended)
 
-API Keys are the recommended authentication method. They can be used when authenticating against the Elasticsearch Service (ESS) or Elastic Cloud Enterprise (ECE).
+API Keys are the recommended authentication method. They can be used to authenticate against Elasticsearch Service or Elastic Cloud Enterprise.
 
-They can either be hardcoded in the provider `.tf` provider configuration (NOT RECOMMENDED). Or specified via environment variables: `EC_API_KEY`.
+They can either be hardcoded in the provider `.tf` configuration (not recommended), or specified with the environment variable `EC_API_KEY`.
```

Comment:
```suggestion
They can either be specified with the `EC_API_KEY` environment variable (recommended) or hardcoded in the provider `.tf` configuration file (supported but not recommended).
```

I would put the recommended option first and mention the other option you can use at the end.